packages/zero-runtime/README.md

```diff
@@ -18,6 +19,26 @@ A zero-runtime CSS-in-JS library that extracts the colocated styles to their own
   - [Color schemes](#color-schemes)
   - [Switching color schemes](#switching-color-schemes)
   - [TypeScript](#typescript)
+- [How-to guides](#how-to-guides)
+  - [Coming from Emotion or styled-components](#coming-from-emotion-or-styled-components)
```

Comment:
```suggestion
  - [Migrating from Emotion or styled-components](#coming-from-emotion-or-styled-components)
```

Maybe that's more obvious?

Reply:
I think I'd stick to `Coming` than `Migrating`. This section is to show some differences in term of mental model with examples, not a docs on how to migrate from emotion to zero.

Reply:
I got it from https://styled-components.com/docs/basics#coming-from-css

Reply:
Makes sense — all good.